Firestore/Example/Tests/Integration/FSTStreamTests.m

```diff
@@ -38,17 +38,25 @@ - (void)writesFinishedWithError:(NSError *_Nullable)error;
  */
 @interface FSTStreamStatusDelegate : NSObject <FSTWatchStreamDelegate, FSTWriteStreamDelegate>
 
+- (instancetype)initFrom:(XCTestCase *)testCase
+              usingQueue:(FSTDispatchQueue *)dispatchQueue NS_DESIGNATED_INITIALIZER;
+- (instancetype)init NS_UNAVAILABLE;
+
 @property(nonatomic, readonly) NSMutableArray<NSString *> *states;
```

Comment:
Consider splitting out implementation properties into a separate interface with an empty category (as is our convention elsewhere). As it stands it's not clear which of these are implementation state and which are meant to be used from within tests that use this delegate.

Reply:
Done. The only public-facing property was 'invokeCallbacks', but I am actually not using it in this PR (and have removed it from the other one as well).